rest_api/src/main/resources/static/assets/js/login.js

```diff
@@ -1,5 +1,5 @@
-var base_api_url = "http://192.168.160.63:8080";
-//var base_api_url = "http://localhost:8080";
+//var base_api_url = "http://192.168.160.63:8080";
```

Comment:
Same problem mentioned before. Here you can either make a commit fixing it or fixing it when anyone solves the existing conflicts with the master